lib/OpenQA/Schema/Result/ScheduledProducts.pm

```diff
@@ -275,6 +273,23 @@ sub _schedule_iso {
             my $prio = delete $settings->{PRIO};
             $settings->{_GROUP_ID} = delete $settings->{GROUP_ID};
 
+            # Any setting name ending in _URL is special: it tells us to download
+            # the file at that URL before running the job
+            my $download_list = create_downloads_list($settings);
+            foreach my $url (keys %$download_list) {
+                my $download_parameters = $download_list->{$url};
+
+        # caveat: The extraction parameter is currently not processed per destination.
+        # If multiple destinations for the same download have a different 'do_extract' parameter the first one will win.
```

Comment:
I suppose tidy caused the wrong indentation. One could likely fix it by wrapping the comment text manually.